### DISCUSSION TITLE
Version 1.9.0 - Integrated downloads, AppleScript

### DISCUSSION THREAD

**Discussion by joelekstrom** [joelekstrom/fastmate#91] [General]
* In-app download support #82
Fastmate now downloads directly to a folder (which can be changed in settings) instead of opening an external browser. The old behavior can still be enabled in settings.
* Add basic AppleScript support #81
* Add support for opening Fastmail `https://` links #88
* Fix some shortcuts not working with new Fastmail layout #70

Extra thanks to @mdbraber for all the work on this release!

<hr /><em>This discussion was created from the release <a href='https://github.com/joelekstrom/fastmate/releases/tag/'>Version 1.9.0 - Integrated downloads, AppleScript</a>.</em>

**Comment by czottmann** [comment]
Thanks for all your work @joelekstrom, I really appreciate it!
